index.html

```diff
@@ -103,11 +105,15 @@ <h5 class="experience__container__content">
         </h5>
         <p class="experience__container__date">December 2023 - March 2024</p>
         <p class="experience__container__description">
-          - Write performant and maintainable code. <br />
-          - Communicate and work with multi-disciplinary team of UI designer, UX
+          <img class="experience__container__icon-triangle" src="./img/svg/icon-triangle.svg"> </>
```

Comment:
Nie dodajemy tutaj specjalnie elementu w HTML, bo możemy to ograć za pomocą `before` lub `after`

Reply:
Komentarz dotyczy całej sekcji